spec/models/case_spec.rb

```diff
@@ -213,22 +213,6 @@
   end
 end
 describe 'scopes', versioning: true do
-  it 'returns cases based on case' do
```

Comment:
@rlgreen91 Was this removed because it's testing Rails functionality (scopes)?

Reply:
@trystant No, I removed it because I removed the scope - we no longer use it anywhere.